templates/user/user_layout.html.twig

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html.twig" %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('css/user') }}
+{% endblock %}
+
+{% block body %}
+<div class="container-fluid bg-light">
+    <div class="container">
+        <div class="position-relative pt-5">
+            <div class="position-absolute lr-100">
+                {% for message in app.flashes('info') %}
+                    <div class="alert alert-info text-center">
+                        {{ message }}
+                    </div>
+                {% endfor %}
```

Comment:
It could be better to make a template we will include to handle all flashes types.

Reply:
IMHO, this is fine for know. we'll see how to factorize code, only when needed